packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -204,7 +204,7 @@ export class DetailsHeaderBase extends React.Component<IDetailsHeaderBaseProps,
                     children: (
                       <DetailsRowCheck
                         id={`${this._id}-check`}
-                        aria-label={ariaLabelForSelectionColumn}
+                        aria-label={ariaLabelForSelectAllCheckbox}

```

Comment:
Since this header is only a select all checkbox in multiple selection mode (and only a header for single selection mode), should we do this instead?

```tsx
aria-label={selectionMode === SelectionMode.multiple ? ariaLabelForSelectAllCheckbox : ariaLabelForSelectionColumn
```

Reply:
Good catch! If the checkbox is rendered with the corresponding `aria-label` for both scenarios then yes we should make it conditional.

@aneeshack4 can you verify?

Reply:
that works yes verified:
![image](https://user-images.githubusercontent.com/4161791/65730078-72016400-e075-11e9-87e0-68dce116fa74.png)

Reply:
Oops I accidentally added the automerge label before pushing the latest commit - I'll make another PR to fix this.

Reply:
PR #10641